src/components/SellNFT.js

```diff
@@ -138,7 +125,9 @@ export default function SellNFT () {
                 </div>
                 <br></br>
                 <div className="text-green text-center">{message}</div>
-                <button onClick={listNFT} className="font-bold mt-10 w-full bg-purple-500 text-white rounded p-2 shadow-lg">
+                <button disabled={loading} 
```

Comment:
This button, or perhaps the page, could use a loading component (meaby an spinner) since tha transaction takes a while.

Reply:
ok.